src/pathfinding/PathfinderResultPrinter.cpp

```diff
@@ -40,6 +41,22 @@ std::string PathfinderResultPrinter::PrintCoordinates(HexPlanet &planet, const P
   return ss.str();
 }
 
+
+std::vector<std::pair<double, double>> PathfinderResultPrinter::GetVector(HexPlanet &planet,
+                                                                          const Pathfinder::Result &result) {
+  std::vector<std::pair<double, double>> pathResult;
+
+  for (HexVertexId id : result.path) {
+    const auto &coord = planet.vertex(id).coordinate;
+    auto lon_str = coord.to_string_longitude();
+    auto lat_str = coord.to_string_latitude();
+
+    pathResult.push_back(std::make_pair(std::stod(lon_str), std::stod(lat_str)));
```

Comment:
Is having the order be lon, lat the convention? Just want to confirm so we can update our local_pathfinding code to match :) no preference on my end.

Reply:
Good catch! I think the order of lat/lon should be switched here. 
When setting the waypoints in the network table, we assume (lat, lon) convention.

However, it does seems like global_pathfinding follows (lon, lat) throughout their code. So, we should probably come to a consensus amongst the 3 groups haha

Reply:
Great, I'll change this to (lat, lon) and then get this merged in.

Sorry for that confusion. The `.kml` convention is (lon,lat) for visualization purposes on Google Earth, so we need it in that order for that, but I like the latlon convention and it sounds like that makes more sense :) thanks for confirming.